docs/editor/extension-gallery.md

```diff
@@ -23,31 +23,32 @@ This will show you a list of the most popular VS Code extensions on the [VS Code
 
 ![popular extensions](images/extension-gallery/extensions-popular.png)
 
-Each extension in the list includes a brief description, the publisher, the download count and a five star rating. You can click on the extension item to display the extension's [VS Code Marketplace](https://marketplace.visualstudio.com/VSCode) page where you can learn more.
+Each extension in the list includes a brief description, the publisher, the download count and a five star rating. You can click on the extension item to display the extension's details page where you can learn more.
```

Comment:
not sure if we want to be sure to say that this comes from the market place page here.

Reply:
I think this is fine. We provide plenty of links to the Marketplace website so users will likely notice that the content is the same.